vagrant/post-deploy.d/20-setup-basic-auth.sh

```diff
@@ -2,6 +2,7 @@
 set -u
 set -e
 echo "Setting up concourse basic auth as ${CONCOURSE_ATC_USER}: ${CONCOURSE_ATC_PASSWORD}"
+sed "s/--external-url.*/--external-url \'\' \\\/" -i /var/vcap/jobs/atc/bin/atc_ctl # Remove hardcoded auth redirect IP
```

Comment:
_what_ upstream project? We only think this AMI is coming from https://github.com/concourse/concourse-lite

Apparently they have some jenkins job that builds the AMI. They have said earlier that AMI is not supported. I am not sure the same issue is present in the lite version - probably not, as this IP could be coming from there.